### DISCUSSION TITLE
Cleaned Data

### DISCUSSION THREAD

**Discussion by sebastianecheverrir** [janosh/matbench-discovery#83] [Q&A] [answered]
Is there any place where the "cleaned data" can be downloaded? 

By "cleaned data" I mean the data that is compatible with the data from the materials project,  resulting from the "Data Processing Steps" described [here](https://github.com/janosh/matbench-discovery/tree/main/data/wbm#--data-processing-steps).

**Comment by janosh** [comment, marked as answer]
have a look at https://matbench-discovery.materialsproject.org/contribute#--direct-download which lists all the MBD data files available on figshare. sounds like you want [2023-12-13-wbm-summary.csv.gz](https://figshare.com/ndownloader/files/44225498)
